tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/largemessages/InterruptedAMQPLargeMessage.java

```diff
@@ -147,7 +147,7 @@ public void testInterruptedLargeMessage() throws Exception {
       AmqpReceiver receiver = session.createReceiver(getQueueName());
 
       int received = 0;
-      receiver.flow((int) (messageCount + 10));
+      receiver.flow((messageCount + 10));
```

Comment:
Can also lose the second set of parentheses that were needed for the cast that was removed.

Reply:
Fixed.